documentation/library_guide/parallel_api/pass_data_algorithms.rst

```diff
@@ -4,8 +4,8 @@ Pass Data to Algorithms
 You can use one of the following ways to pass data to an algorithm executed with a device policy:
 
 * ``oneapi:dpl::begin`` and ``oneapi::dpl::end`` functions
-* Unified shared memory (USM) pointers and ``std::vector`` with USM allocators
-* Iterators of host-side ``std::vector``
+* Unified shared memory (USM) pointers
+* Iterators or USM pointers to ``std::vector`` data
```

Comment:
I guess the phrase `Iterators or USM pointers to ``std::vector`` data` has a double meaning...
I think it is better to write it more clearly, in two bullets:

- Iterators of host-side ``std::vector``
- USM pointers obtained as ``std::vector::data()`` and ``std::vector::data() + std::vector::size()`` in case of USM allocator is passed into ``std::vector`` constructor.

Reply:
Thanks, I've applied this change.